app/models/order.rb

```diff
@@ -15,22 +15,35 @@ def total_quantity
     item_orders.sum("quantity")
   end
 
-  def change_to_cancel_status 
+  def change_to_cancel_status
     update(status: 3)
   end
 
-  def unfulfill_item_orders 
-    item_orders.each do |item_order| 
-      if item_order.fulfilled? 
+  def unfulfill_item_orders
+    item_orders.each do |item_order|
+      if item_order.fulfilled?
         item_order.update(fulfilled?: false)
         item = Item.find(item_order.item_id)
         item.update(inventory: (item.inventory + item_order.quantity))
       end
     end
   end
 
-  def cancel 
+  def cancel
     change_to_cancel_status
     unfulfill_item_orders
   end
+
+  def package_order
```

Comment:
Awesome job. I know this is currently Ruby - if we ever get a chance to look back and refactor, I think we could tackle the Active Record now